### PR TITLE
chore: Do not keep a lock file for highest deps resolution

### PR DESCRIPTION
**Pull request by edgarrmondragon**
## Checklist

- [x] My pull request has a descriptive title.
- [x] I have read the [CONTRIBUTING] guide.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] If appropriate, I have added necessary documentation.
- [ ] For user-facing changes, refactorings, performance improvements or documentation updates, I have added a changelog entry using [Changie].

For both the title of the PR and the changelog entry, prefer simple past tense or constructions with "now". For example:

  - Added `Client.invite_participants()`
  - `Client.user_activation_settings()` now accepts a `user_activation_settings` keyword argument

[CONTRIBUTING]: https://citric.readthedocs.io/en/latest/contributing/code-of-conduct.html
[Changie]: https://changie.dev/

----
📚 Documentation preview 📚: https://citric--1143.org.readthedocs.build/en/1143/

### CODE REVIEW

**Comment by edgarrmondragon** [comment]
Ping @GabxTaro

File: docs/requirements.txt [hunk 1]
```diff
@@ -64,7 +64,7 @@ pyyaml==6.0.1
     # via
     #   myst-parser
     #   sphinx-autoapi
-requests==2.31.0
+requests==2.32.0
```
Comment (edgarrmondragon) [inline docs/requirements.txt:67]:
```suggestion
requests==2.31.0
```

File: docs/requirements.txt [hunk 2]
```diff
@@ -64,7 +64,7 @@ pyyaml==6.0.1
     # via
     #   myst-parser
     #   sphinx-autoapi
-requests==2.31.0
+requests==2.32.1
```
Comment (edgarrmondragon) [inline docs/requirements.txt:67]:
```suggestion
requests==2.32.0
```

File: docs/requirements.txt (hunk 1 above)
Comment (edgarrmondragon) [inline docs/requirements.txt:67]:
```suggestion
requests==2.32.1
```

**Review by coeurdestenebres — APPROVED** [review]

**Pull request merged**